File: 백준/Bronze/2525. 오븐 시계/오븐 시계.py
h, m = map(int, input().split())
t = int(input())
if(m+t>60):
    x = (m+t)%60
    y = (m+t)//60
    h = h+y
    if(h>23):
        h = h-24
        print(h, x)
    else:
        print(h, x)
elif(m+t==60):
    x = (m+t)%60
    y = (m+t)//60
    h = h+y
    if(h>23):
        h = h-24
        print(h, x)
    else:
        print(h, x)
else: 
    print(h, m+t)

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        String[] input = br.readLine().split(" ");
        int n1 = Integer.parseInt(input[0]);
        int n2 = Integer.parseInt(input[1]);
        int n3 = Integer.parseInt(br.readLine());

        int seconds = (n2+n3)/60;
        int minutes = (n2+n3)%60;
        int hours = n1+ seconds;

        if(hours>23){
            hours= hours-24;
        }
        bw.write(hours + " " + minutes);

        bw.flush();
        bw.close();


    }
}
